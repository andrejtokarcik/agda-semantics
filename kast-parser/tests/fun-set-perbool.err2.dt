data Bool : Set where
  True : Bool,
  False : Bool
;
data T : Set where
  E : T

; s True = Bool
; s False = T
; f = Lambda x (s True) x
; g = f True
; h = f E
