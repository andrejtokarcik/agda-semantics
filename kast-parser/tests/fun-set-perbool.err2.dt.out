<dbg>
    .List
</dbg>
<T>
    <k>
        ( s True , #symExpr(4) ) = ( Bool , Set 0 ) ~> funDecl ( s , .Map 
          ) ~> s False = T ~> f = (Lambda x s True x) ~> g =  f True  ~> h =  
          f E 
    </k>
    <mgu>
        subst(.KList)
    </mgu>
    <data>
        Bool |-> Set 0
        E |-> T
        False |-> Bool
        T |-> Set 0
        True |-> Bool
    </data>
    <meta>
        .Map
    </meta>
    <fun>
        .Map
    </fun>
    <tenv>
        s |-> (Pi #symVariable(2) Bool #symExpr(4))
    </tenv>
</T>
